tests/last_argument_expansion/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -109,19 +109,21 @@ true
   ]
 });
 
-true
-  ? test({
-      a: 1
-    })
-  : <div
-      a={123412342314}
-      b={123412341234}
-      c={123412341234}
-      d={123412341234}
-      e={123412341234}
-      f={123412341234}
-      g={123412341234}
-    />;
+true ? (
+  test({
+    a: 1
+  })
+) : (
```

Comment:
I used to like this format because it works well with editors and manual formatting, but now that prettier handles formatting I ended up preferring the prettier format. It is "lighter" and still unambiguous.